circle.yml

```
@@ -32,12 +32,7 @@ jobs:
       - run:
           name: Test
           command: opam config exec -- make coverage
-      - run:
-          name: Install ocveralls
-          command: opam install -y ocveralls
       - run:
           name: Upload coverage report
           command: |
-            cd _build/default
-            shopt -s globstar
-            opam config exec -- ocveralls **/bisect*.out --send --repo_token $COVERALLS_REPO_TOKEN --git
+            opam config exec -- bisect-ppx-report --send-to Coveralls
```

```suggestion
            opam config exec -- bisect-ppx-report send-to Coveralls
```

This was suggested by the maintainer on the pgx repository